spec/models/manageiq/providers/ibm_power_vc/cloud_manager/refresher_spec.rb

```diff
@@ -2,6 +2,23 @@
   let(:zone) { EvmSpecHelper.create_guid_miq_server_zone.last }
   let(:ems) { FactoryBot.create(:ems_ibm_power_vc_with_vcr_authentication, :zone => zone) }
 
+  specs = YAML.load_file(File.join(__dir__, 'state.yml'))
+  specs['resources'].each do |key, _value|
+    case key['type']
+    when 'openstack_compute_flavor_v2'
+      let(:flavorid) { key['instances'][0]['attributes']['id'] }
+    when 'openstack_identity_project_v3'
+      let(:tenantid) { key['instances'][0]['attributes']['id'] }
+    when 'openstack_compute_instance_v2'
+      let(:instid) { key['instances'][0]['attributes']['id'] }
```

Comment:
If we end up creating more than one instance we'll have to be more specific since this would end up taking the last instance in the list.

Reply:
yes definitely. We are checking for just one VM right now. We will have to revisit it once we enhance our test specs.